tapir/settings.py

```diff
@@ -239,3 +255,20 @@
     SILKY_PYTHON_PROFILER = True
     SILKY_PYTHON_PROFILER_BINARY = True
     SILKY_META = True
+
+if ENABLE_API:
+    REST_FRAMEWORK = {
+        # Use Django's standard `django.contrib.auth` permissions,
+        # or allow read-only access for unauthenticated users.
+        "DEFAULT_PERMISSION_CLASSES": [
+            "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly"
+        ],
+        "DEFAULT_AUTHENTICATION_CLASSES": (
+            "rest_framework_simplejwt.authentication.JWTAuthentication",
+        ),
+    }
+
+    SIMPLE_JWT = {
+        "ACCESS_TOKEN_LIFETIME": timedelta(minutes=10),
+        "REFRESH_TOKEN_LIFETIME": timedelta(weeks=12),
```

Comment:
Does this mean they have to re-login to the app every 12 weeks?

Reply:
They only have to re-login if they haven't used the app for 12 weeks. The app uses the refresh token to obtain a new access token when the access token has expired (10 minutes). This way, you can disable a refresh token to effectively force a fresh login.